_rules/80f0bf.md

```diff
@@ -0,0 +1,145 @@
+---
+id: 80f0bf
+name: Video or audio has no auto-play audio.
+rule_type: composite
+description: |
+    This rule checks that auto-play audio does not last for more than 3 seconds, or the audio has a control mechanism to stop or mute the it.
+accessibility_requirements:
+  wcag20:1.4.2: # Audio Control (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_rules:
+  - [Auto-play audio has a control mechanism](https://act-rules.github.io/rules/95e043)
+  - [Auto-play audio is less than 3 seconds](https://act-rules.github.io/rules/aaa1bf)
+authors:
+  - Anne Thyme Nørregaard
+  - Bryn ANderson
+---
+
+## Applicability
+
+This rule applies to any HTML `<audio>`, `<video>` or `<source>` elements, with a `src` attribute referencing content with a duration of more than 3 seconds, that has an `autoplay` attribute equal to `true`, and that has either a `paused` or `muted` attribute equal to `false`.
```

Comment:
I believe the logic of the applicability is faulty.

They way I read it now this rule is only applicable if there is either a paused or muted attribute that is equal to false.

The first thing I notice is that there is no paused attribute (only a property).

And the logic now translates to:
1. muted FALSE paused FALSE -> applicable
2. muted FALSE paused TRUE -> applicable
3. muted TRUE  paused FALSE -> applicable
4. muted TRUE  paused TRUE -> not applicable

While I do believe that both situation 2 and 3 should be **not** applicable.

I think this "and that has either a `paused` or `muted` attribute equal to `false`" should be "and that has both a 'paused' and 'muted' property equal to 'false'".

Reply:
Thanks for spotting this @ShadowBB , the rule has been updated.